src/lib/logger.ts

```diff
@@ -1,7 +1,9 @@
 import { createLogger } from "@companieshouse/structured-logging-node";
 import ApplicationLogger from "@companieshouse/structured-logging-node/lib/ApplicationLogger";
+import { PSC_VERIFICATION_WEB_NAMESPACE } from "./constants";
+import { env } from "../config";
 
-export const logger: ApplicationLogger = createLogger(process.env.APP_NAME ?? "");
+export const logger: ApplicationLogger = createLogger(env.APP_NAME ?? PSC_VERIFICATION_WEB_NAMESPACE);
 
 // tslint:disable-next-line:no-console
 console.log(`env.LOG_LEVEL set to ${process.env.LOG_LEVEL}`);
```

Comment:
I suppose just to match, the `process` can be dropped from this console log too, to leave `${env.LOG_LEVEL}`

Reply:
Good spot - thanks!